src/main/java/io/supertokens/inmemorydb/Start.java

```diff
@@ -1619,4 +1627,86 @@ public void addInfoToNonAuthRecipesBasedOnUserId(String className, String userId
             throw new IllegalStateException("ClassName: " + className + " is not part of NonAuthRecipeStorage");
         }
     }
+
+    // TOTP recipe: 
+
+    @Override
+    public void createDevice(TOTPDevice device) throws StorageQueryException {
+        try {
+            TOTPQueries.createDevice(this, device);
+        } catch (Exception e) {
```

Comment:
Never catch all exceptions. See how other queries are done in this PR

Reply:
Done.